tests/unit/type/zset.tcl

```diff
@@ -1506,6 +1506,7 @@ start_server {tags {"zset"}} {
         }
 
         test "ZSCORE - $encoding" {
+            set tcl_precision 17
```

Comment:
i see docs says `tcl_precision` is deprecated since 8.5 (our minimum version), and that format using `%.17` is recommended instead.
why do you need it?

Reply:
no need. removed